bundles/org.openhab.binding.unifiedremote/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="unifiedremote"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="server">
+		<label>Unified Remote Server</label>
+		<description>Unified Remote Server Thing for Unified Remote Binding</description>
+		<channels>
+			<channel id="mouse-move" typeId="mouse-move-channel"/>
+			<channel id="send-key" typeId="send-key-channel"/>
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Hostname</label>
+				<description>Unified Remote Server Hostname</description>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.